cmd/cmd.go

```diff
@@ -939,6 +1036,35 @@ func RunServer(cmd *cobra.Command, _ []string) error {
 	return server.Serve(ln, origins)
 }
 
+func getBase64Image(filePath string) (string, error) {
+	_, err := os.Stat(filePath)
+	if err != nil {
+		fmt.Printf("Couldn't find image %s\n", filePath)
+		return "", err
+	}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		return "", err
+	}
+	defer file.Close()
+
+	data, err := io.ReadAll(file)
```

Comment:
`http.DetectContentType` uses at most [512 bytes](https://pkg.go.dev/net/http#DetectContentType) to determine content type. It might be a good idea to read at most 512 B here, run detect, then consume the rest of the file is it's an image type